CentralComputing/Utils.cpp

```diff
@@ -40,20 +41,20 @@ void Utils::print(LogLevel level, const char * format, ...){
       }
 }
 
-int Utils::toogleGPIOUp(int GPIONumber) {
-    std::string start = "/sys/class/gpio/gpio"
+int Utils::set_GPIO_high(int GPIONumber) {
```

Comment:
I changed the name from `toggle` to `set`. `toggle` usually implies switching the output, i.e. `toggle_GPIO(GPIO_number)` would figure out which state the output is in, and then flip it. You don't toggle a pin to high, you set it to high.

Just being pedantic